lib/FrontendTool/FrontendTool.cpp

```diff
@@ -99,6 +99,44 @@ static std::string displayName(StringRef MainExecutablePath) {
   return Name;
 }
 
+namespace swift {
+namespace frontend {
+namespace utils {
+StringRef quoteForMake(StringRef raw, llvm::SmallVectorImpl<char> &buffer) {
```

Comment:
Nitpick: can you do this with `::` instead of nested namespaces?

Reply:
Also, maybe `escapeForMake` to go with `llvm::yaml::escape`, especially since we're not using quotes.

Reply:
Sure, both are fine by me; will do.